src/frontend/components/InstructorView/index.tsx

```diff
@@ -22,6 +22,12 @@ const messages = defineMessages({
       'Text explaining that the ivdeo is in read_only mode and the dashboard is not available',
     id: 'components.InstructorView.disabledDashboard',
   },
+  maintenance: {
+    defaultMessage: "The dashboard is in maintenance, you can't access it",
```

Comment:
```suggestion
    defaultMessage: "The dashboard is undergoing maintenance work, it can't be accessed right now.",
```

Maybe this is a little clearer?

Reply:
:+1: